### PR TITLE
[DAT-9954] Created tests for EDB with EDB driver

### PR DESCRIPTION
**Pull request by yodzhubeiskyi**

### CODE REVIEW

File: src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.6/alterSequence.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.6/alterSequence.json:1]: can liquibase get more details about sequences for edb<10? This json doesn't actually validate if sequence was altered - like `incrementBy` `cycle` or other fields changed something. Default (liquibase/harness/change/expectedSnapshot/alterSequence.json) does more precise checks

File: src/main/resources/liquibase/harness/change/expectedSql/edb-e/enableCheckConstraint.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+INVALID TEST
+
+enableCheckConstraint is not supported for EDB
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/change/expectedSql/edb-e/enableCheckConstraint.sql:3]: same. we don't have EDB in docs (at least for now). and plz remove dashes

**Review by KushnirykOleh — CHANGES_REQUESTED** [review]
During meeting we spoke how to call EDB with EDB driver comparing to EDB with Postgres driver that we had before. One of ideas was edb-edb. Or maybe @kristyldatical has better ideas. edb-e is not meaningful name as for me.

File: src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.6/alterSequence.json (hunk 1 above)
Comment (yodzhubeiskyi) [reply src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.6/alterSequence.json:1]: I'll need to take a closer look at this issue. Perhaps additional core functionality can be added.

Comment (kristyldatical) [reply src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.6/alterSequence.json:1]: After discussion at standup today, we will go with `edb-edb` for EDB with the EDB driver and `edb-postgres` for EDB with the postgres driver. CC @yodzhubeiskyi

File: src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.5/alterSequence.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "snapshot": {
+    "objects": {
+      "liquibase.structure.core.Sequence": [
+        {
+          "sequence": {
+            "name": "test_sequence"
```
Comment (AlexanderSashchenko) [inline src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.5/alterSequence.json:7]: add "body" attribute please

**Review by AlexanderSashchenko — CHANGES_REQUESTED** [review]

File: src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.5/alterSequence.json (hunk 3 above)
Comment (yodzhubeiskyi) [reply src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.5/alterSequence.json]: Liqubase snapshot doesn't capture more information for EDB versions 9.5 and 9.6. Same behavior is true for EDB with Postgres driver

**Comment by nvoxland** [comment]
I switched the base on this to "develop" and renamed the branch on the liquibase-core side to match this so that this builds using the fixes in liquibase/liquibase#2767 vs. master

**Pull request merged**